### ISSUE TITLE
type "dead" characters actually set Dead in the textbox

### ISSUE THREAD

**Issue by julienGrd** [RobinHerbots/jquery.inputmask#2701]
Hello, i notice when i associate textbox to a mask with your library, when i type a dead character (dead character are actually character who need to be associate with another one, like ^ or ¨), it will literally write Dead in the field (the behavior expected is nothing happen, and it will be the next character pressed which will be associate with the dead character).

In french, we use it for a lot of word (noël or crême for example)

so basically when i want to write crême it will write crDeademe in the textbox

you can reproduce it on your demo on the field email address :https://robinherbots.github.io/Inputmask/#/demo
type a dead character in the field, you will see Dead appear

Thanks !

**Comment by RobinHerbots** [comment]
@julienGrd ,

I fixed the issue in  version 5.0.9-beta.1 (no npm release yet)

**Comment by sjacq** [comment]
Hi, We updated from 4.x where the accents were working,  but there's still a minor issue with those characters. With 5.0.9-beta.1, the DEAD word is not inserted anymore, but instead of getting the accented character like **é** in café, you get the accent and then the letter. So you end up with **caf'e**.

Thanks.

**Comment by RobinHerbots** [comment]
@sjacq ,

Can you have a retry with the latest beta.

I tried something like 
```
	Inputmask({
		mask: "a{*}"
	}).mask("#test1");
```
and i could type élève and café
